lib/gooddata/models/domain.rb

```diff
@@ -432,7 +439,14 @@ def synchronize_clients
     # Runs async process that walks through segments and provisions projects if necessary.
     #
     # @return [Enumerator] Returns Enumerator of results
-    def provision_client_projects(segments = nil)
+    def provision_client_projects(options = {})
+      if options[:data_products] == :all
```

Comment:
Refers to 'options' more than self (maybe move it to another class?) (FeatureEnvy)